spec/fixtures/custom_nginx.template

```diff
@@ -503,6 +503,17 @@ http {
                 Kong.serve_cluster_listener()
             }
         }
+        location = /version-handshake {
+            limit_except POST {
+                deny all;
+            }
+            if ($content_type != "application/json") {
+                return 400;
+            }
+            content_by_lua_block {
+                Kong.serve_version_handshake()
+            }
+        }
```

Comment:
mostly to share the same port and SSL certificates as the websocket endpoint.  otherwise we would need another config option to specify how to connect to the negotiation endpoint.

of course, this `.serve_version_handshake()` could be implemented with lapis, but i don't see any advantage.